PhysicsTools/PatAlgos/python/slimming/prunedGenParticles_cfi.py

```diff
@@ -15,6 +15,7 @@
         "drop   status == 2 && abs(pdgId) == 21",                          # but remove again gluons in the inheritance chain
         "keep abs(pdgId) == 23 || abs(pdgId) == 24 || abs(pdgId) == 25 || abs(pdgId) == 6 || abs(pdgId) == 37 ",   # keep VIP(articles)s
         "keep abs(pdgId) == 310 && abs(eta) < 2.5 && pt > 1 ",                                                     # keep K0
+        "+keep abs(pdgId) == 13 && status == 1", # keep muon parents
```

Comment:
``` "++keep abs(pdgId) == 11 || abs(pdgId) == 13 || abs(pdgId) == 15" ``` above is seemingly more inclusive and should have picked up all muons and their parents.
Am I missing some "drop" or is the "++keep abs(pdgId) == 13 " syntax not that inclusive and misses to store muons?